static/app/views/settings/project/projectOwnership/editRulesModal.tsx

```diff
@@ -27,7 +27,9 @@ class EditOwnershipRulesModal extends Component<Props, State> {
         <Block>
           {t('Examples')}
           <CodeBlock>
-            path:src/example/pipeline/* person@sentry.io #infrastructure
```

Comment:
why the change from `#infrastructure` to `#infra`

Reply:
It's `#infra` in two other places